README.md

```diff
@@ -32,4 +32,125 @@ The built in compiler remains solc 0.6, but users who want to use the newer comp
 ```
 npm install -D solc-0.7@npm:solc@^0.7.0
 npx solidity-docgen --solc-module solc-0.7
+```
+
+
+## Example Handlebars Formatting
+
+Each of the following examples will go through various methods exposed by the solidity-docgen library, how to use them in handlebars, and what the resulting markdown output would look like.
+
+1. Getting the natspec @dev and @notice tags
```

Comment:
This is getting rendered as a numbered list and I think we want them to look more like headings, so I would go with:
```suggestion
### 1. Getting the natspec @dev and @notice tags
```

And we should increase the nesting of the two subsections in each of these.